index.js

```diff
@@ -175,17 +177,17 @@ module.exports = function (config) {
    */
 
   // serve all the provider json
-  app.get('/providers', function (req, res, next) {
-    res.json(app.services)
+  koop.get('/providers', function (req, res, next) {
+    res.json(koop.services)
   })
 
   // serve up a provider
-  app.get('/providers/:provider', function (req, res, next) {
-    res.json(app.services[req.params.provider])
+  koop.get('/providers/:provider', function (req, res, next) {
```

Comment:
I would say we should keep these here. It's pretty hard to see what a koop instance is running from the outside without these. If anything I'd like to combine this and `status` which isn't really used into something informative for any koop instance. If koop-ui consumes it that's a bonus but I think it's more useful as an API endpoint to check health, status, versions, etc.